clang/test/Driver/riscv-cfi-property.c

```diff
@@ -0,0 +1,29 @@
+// When -march with zicfiss0p4 or zicfilp0p4 add GNU property to file object
```

Comment:
This directory is for driver tests. Codegen tests should use `%clang_cc1` and be moved elsewhere (e.g. test/CodeGen)

Reply:
I should only test in llvm-readobj not in Driver
Thanks